src/plan/gencopy/global.rs

```diff
@@ -172,11 +172,8 @@ impl<VM: VMBinding> Plan for GenCopy<VM> {
     }
 
     fn global_side_metadata_per_chunk(&self) -> usize {
-        if !VM::VMObjectModel::HAS_GC_BYTE {
-            meta_bytes_per_chunk(3, 1)
-        } else {
-            0
-        }
+        debug_assert!(VM::VMObjectModel::HAS_GC_BYTE);
+        meta_bytes_per_chunk(3, 0)
```

Comment:
Can you explain this change? Previously the plan needed 2 bits global side metadata if we don't have GC bytes, and needed 0 if we have GC bytes. Now the plan needed 1 bit and assumed we always have GC bytes.  Is the 1 bit for `LOGGING_META`? What's the reason we need to assume we always have GC bytes for GenCopy?

Reply:
If openjdk turns off gc-byte support, then the gc byte has to be moved to side-metadata, which means the barrier also needs to be updated to adapt to the new layout... So I just add an assertion here to check this can never happen.

Reply:
If it is for openjdk, you probably need to keep the assertions at the openjdk side.

Reply:
Done. The check is now moved to openjdk binding